content/1_General/Expected.problems.json

```
@@ -13,6 +13,19 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "cf-231A",
+      "name": "Team",
+      "url": "https://codeforces.com/problemset/problem/231/A",
+      "source": "CF",
+      "difficulty": "Very Easy",
+      "isStarred": false,
+      "tags": ["Brute Force", "Greedy"],
+      "solutionMetadata": {
+        "kind": "autogen-label-from-site",
+        "site": "CF"
```

change kind to internal
```suggestion
        "kind": "internal"
```